FlickTrainer/app/src/main/java/jp/co/azz/flicktrainer/databases/HistoryDao.java

```diff
@@ -0,0 +1,200 @@
+package jp.co.azz.flicktrainer.databases;
+
+import android.content.ContentValues;
+import android.content.Context;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+public class HistoryDao {
+    private DatabaseHelper dataBaseHelper;
+
+    public HistoryDao(Context context) {
+        dataBaseHelper = new DatabaseHelper(context);
+    }
+
+    //select文のOrderby句に指定する、TOP5件分を取得するための定数
+    private final String TOP_LIMIT5 = " DESC LIMIT 5";
+
+    /**
+     * History テーブルよりスコアTop5 を返却するSELECT文を発行する。
+     *
+     * @return HistoryDto型でSelectされた値を返す。
+     */
+    public List<HistoryDto> selectHistoryScoreTop5() {
```

Comment:
自分のランキングデータを取得することが分かるメソッド名にしましょう。
```suggestion
    public List<HistoryDto> getRanking() {
```

Reply:
メソッド名をgetRankingに変更しました。